packages/gatsby/src/commands/develop.ts

```diff
@@ -260,7 +280,7 @@ async function startServer(program) {
             .on(`response`, response =>
               res.writeHead(response.statusCode, response.headers)
             )
-            .on(`error`, (err, _, response) => {
+            .on(`error`, (err: Error, _: any, response) => {
```

Comment:
Random but Flow has a `mixed` type. Basically means "don't know what's in it, I'm not touching it, so whatever". Does TS have this as well? Ultimately it's an `any` but the semantics are a little different and the linting can make a distinction.

Reply:
(Not reviewing, just caught my eye and curious what the TS way is here, if anything)

Reply:
@pvdz i like the thought behind this. Unfortunately typescript doesn't. We could implement our own easily if we wanted a mixed type `declare mixed = any;`.

I did go and update this though, ideally we should be typing out everything and not using any. this was coming from the library `got` so i just added `@types/got` and now these types are properly typed out for us. So good flag. This is an example of how we should strive to not use loose types.

Reply:
TS has. You can use `unknown` or `never` here and I personally recommend `never`